src/WebJobs.Script.WebHost/ContainerManagement/ILinuxFunctionExecutionActivityPublisher.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.ContainerManagement
+{
+    public interface ILinuxFunctionExecutionActivityPublisher
```

Comment:
Wonder if this shouldn't be named ILinuxContainerActivityPublisher? For now it'd just have PublishFunctionActivity